package.json

```diff
@@ -35,6 +35,7 @@
     "react-dom": "^17.0.1",
     "react-redux": "^7.2.2",
     "react-table": "^7.6.2",
+    "react-tooltip": "^4.2.11",
```

Comment:
This works too but I think I have some small preference for using the tooltips in react-boostrap since we're using that for our other UI components: https://react-bootstrap.github.io/components/overlays/#tooltips